mlrun/db/httpdb.py

```diff
@@ -4358,7 +4360,10 @@ def store_alert_config(
 
         alert_data = alert_instance.to_dict()
         body = _as_json(alert_data)
-        response = self.api_call("PUT", endpoint_path, error_message, body=body)
+        params = {"force_reset": bool2str(force_reset)}
```

Comment:
If false it can be ignored
```suggestion
        params = {"force_reset": bool2str(force_reset)} if force_reset else {}
```